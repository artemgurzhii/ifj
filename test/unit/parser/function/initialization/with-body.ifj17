function foo(bar as string) as string
  return !"hello " + bar
end function
