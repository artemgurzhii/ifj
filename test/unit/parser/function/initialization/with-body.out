(function foo -> (id string)
  (decl
    (id bar)
    : (id string))
  (return
    (+ (string 'hello ') (id bar))))
